streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -257,7 +257,7 @@ class ZStream[-R, +E, +A](val channel: ZChannel[R, Any, Any, Any, E, Chunk[A], A
    */
   final def broadcast(n: Int, maximumLag: Int)(implicit
     trace: ZTraceElement
-  ): ZManaged[R, Nothing, List[ZStream[Any, E, A]]] =
+  ): ZManaged[R, Nothing, Chunk[ZStream[Any, E, A]]] =
```

Comment:
I changed the signature here to return a `Chunk`. I think this is more idiomatic and consistent with our philosophy of using `Chunk` throughout. It is also somewhat more ergonomic because you can index into the streams directly without having to pattern match and deal with the impossible case where the list does not have the expected number of elements.